hledger-dep.json

```diff
@@ -0,0 +1,3386 @@
+{
+  "name": "hledger-exe",
+  "buildsystem": "simple",
+  "build-commands": [],
+  "modules": [
+    {
+      "name": "ghc-9.6.6-aarch64",
+      "only-arches": [
+        "aarch64"
+      ],
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.6.6",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.6.6",
+        "/bin/ghci",
+        "/bin/ghci-9.6.6",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.6.6",
+        "/bin/runghc",
+        "/bin/runghc-9.6.6",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.6.6",
+        "/share/man",
+        "/share/doc/ghc-9.6.6"
+      ],
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/aarch64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
+        "./configure --prefix=/app",
+        "make install"
+      ],
+      "sources": [
+        {
+          "sha256": "58d5ce65758ec5179b448e4e1a2f835924b4ada96cf56af80d011bed87d91fef",
+          "type": "archive",
+          "url": "https://downloads.haskell.org/ghc/9.6.6/ghc-9.6.6-aarch64-deb10-linux.tar.xz"
+        }
+      ]
+    },
+    {
+      "name": "ghc-9.6.6-x86_64",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.6.6",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.6.6",
+        "/bin/ghci",
+        "/bin/ghci-9.6.6",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.6.6",
+        "/bin/runghc",
+        "/bin/runghc-9.6.6",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.6.6",
+        "/share/man",
+        "/share/doc/ghc-9.6.6"
+      ],
+      "buildsystem": "simple",
+      "builddir": false,
+      "build-commands": [
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/x86_64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
+        "./configure --prefix=/app",
+        "make install"
```

Comment:
this was not fixed.

Reply:
applying your suggestion (as I naively understood it) caused the build to fail with an opaque message, and I was debugging that.

what I did was change
```json
"buildsystem": "simple",
"build-commands": [
  "mkdir -p /app/lib",
  "ln -s /usr/lib/aarch64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5",
  "./configure --prefix=/app",
  "make install"
],
```
to
```json
"buildsystem": "autotools",
"post-install": [
  "mkdir -p /app/lib",
  "ln -s /usr/lib/aarch64-linux-gnu/libtinfo.so /app/lib/libtinfo.so.5"
],
```
and I got the error
```
...
****************************************************
Configuration done, ready to 'make install'
(see README and INSTALL files for more info.)
****************************************************
Run "make install" to install
make: *** [Makefile:10: default] Error 1
Error: module ghc-9.6.6-aarch64: Child process exited with code 2
```

Reply:
It's fixed now.